pkg/bgpv1/types/bgp.go

```diff
@@ -17,9 +17,10 @@ const (
 	DefaultBGPConnectRetryTime = 120 * time.Second
 	// DefaultBGPHoldTime defines the default initial value for the BGP HoldTimer (RFC 4271, Section 4.2).
 	DefaultBGPHoldTime = 90 * time.Second
-
 	// DefaultGRRestartTime defines default Restart Time for graceful restart (RFC 4724, section 4.2)
 	DefaultGRRestartTime = 120 * time.Second
+	// DefaultPeerPort is the TCP port number of a CiliumBGPNeighbor when PeerPort is unspecified.
+	DefaultPeerPort = 179
```

Comment:
Commit 11da300  changes this const to untyped based on [this](https://go.dev/blog/constants) guidance.